_posts/2021-02-12-generalist-specialization-and-the-impostor-syndrome.md

```diff
@@ -0,0 +1,80 @@
+
+So I would call myself a generalist. That's been my specialization. Meaning that I actively pursue that and keep my generalization up to date. I know few things really in-depth, but I know a hundred things on a level that I can rapidly refresh up to today's requirements if there's a need. A few years back someone asked me about XSLT templates, so I whipped up from my ancient technologies archive a nice recursion template that lets you do crazy stuff with minimal code. That gives me much flexibility to jump in many places. Unfortunately, that comes with a downside too. A constant visitor.
+
+## How can you deal with the Impostor Syndrome?
+
+There are two bad ways to try to deal with this. One is to never put yourself in a position of change or challenge. Just play it very safe. You learned Java 1.0 in 1996, so keep on using it for everything, and only look for jobs where that is all that is required. This approach would let the Impostor Syndrome rule and limit you. Another is to wait, it will pass. But that's not going to help you deal with it any better next time, so while it may be a victory, it's a lame one.
+
+A better approach is just to acknowledge that it is a thing, and recognize it. Understand that if you feel like that, it does not mean you actually are an impostor. But most of us have studied hard, have practiced hard, have worked hard, more or less. So the background is there, the skills are there. That feeling is not a real reflection of things.
+
+It's also not a mental disorder, nor a permanent state of mind. It's a reaction to stimuli, especially when you are high-performing and reaching far. It could be compared to driving a sports car at high speeds, and suddenly having a doubt if you can control it if you actually have the skills for that. It's a temporary moment of self-doubt, that often happens at the moment when you have achieved (or are about to achieve) an important milestone. New job, new project, a new position, or just praise and adoration for the work you have done. Ironically, this can also be triggered by positive feedback and compliments.
```

Comment:
"suddenly having a doubt if you can control it if you actually have the skills for that." -- is there a word missing here?

Perhaps something like "suddenly starting to doubt whether you can actually control it" .

"self-doubt, that" -> "self-doubt that" (there's almost never a comma before "that")

"happens ~at the moment~ when" -- "at the moment" + "when" == redundant